kpi/deployment_backends/kobocat_backend.py

```diff
@@ -286,7 +286,7 @@ def redeploy(self, active=None):
                 'version': self.asset.version_id,
             })
         except KobocatDeploymentException as e:
-            if e.response.status_code == 404:
+            if hasattr(e, 'response') and e.response.status_code == 404:
```

Comment:
Much cleaner, but it's `response.status_code` that contains the `404`, not `response` itself. This pattern of getting `first.second.third` while dodging an `AttributeError` if `second` doesn't exist seems really common; maybe there's a more Pythonic way than what I wrote?